en/Obsidian Publish/Set up a custom domain.md

```diff
@@ -126,6 +126,33 @@ mysite.com {
 }
 ```
 
+### Traefik
+
+This minimal configuration example redirects `example.com` to Obsidian publish.
```

Comment:
Would it make more sense to keep using `mysite.com` just like the other examples?

Reply:
Oops, missed that. Will do